pubsub/rabbitmq/metadata.go

```diff
@@ -19,6 +19,8 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/dapr/components-contrib/internal/utils"
```

Comment:
this will probably fail the linter because the dapr libraries must be the last import block